src/custom.d.ts

```diff
@@ -1,3 +1,6 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+declare type FixMe = any
```

Comment:
We use this in other projects to mark "unwanted" `any`s. I just swtiched `route.js` to TS and it needs to be improved with exact types.